aptos-node/src/lib.rs

```diff
@@ -288,6 +288,8 @@ where
             genesis_waypoint.to_string().as_bytes(),
         )?;
 
+        aptos_config::config::sanitize_node_config(&mut validators[0].config)?;
```

Comment:
Any reason we don't sanitize all node configs?

Reply:
`--test` mode is only one validator. If that changes this wouldn't work right, but we have `forge` local swarms for things beyond this test mode.